src/Domains/domain.jl

```diff
@@ -133,49 +133,55 @@ function PeriodicPlane(
 end
 
 """
-    Base.ndims(::Column)
-    
-Query function for `Column` domain (number of dimensions)
+    struct HVSpace2D{FT} <: AbstractHybridDomain 
```

Comment:
Let's give this struct a more descriptive name. How `HybridPlane` to go with the other Planes?